Detections/AuditLogs/AzureADRoleManagementPermissionGrant.yaml

```diff
@@ -0,0 +1,55 @@
+id: 1ff56009-db01-4615-8211-d4fda21da02d
+name: Azure AD Role Management Permission Grant
+description: |
+  'Identifies when the Microsoft Graph RoleManagement.ReadWrite.Directory (Delegated or Application) permission is granted to a service principal.
+  This permission allows an application to read and manage the role-based access control (RBAC) settings for your company's directory.
+  An adversary could use this permission to add an Azure AD object to an Admin directory role.
+  Ref : https://docs.microsoft.com/en-us/graph/permissions-reference#role-management-permissions
+  Ref : https://docs.microsoft.com/en-us/graph/api/directoryrole-post-members?view=graph-rest-1.0&tabs=http'
+severity: High
+requiredDataConnectors:
+  - connectorId: AzureActiveDirectory
+    dataTypes:
+      - AuditLogs
+queryFrequency: 2h
+queryPeriod: 2h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - PrivilegeEscalation
+  - Persistence
+relevantTechniques:
+  - T1098.003
+tags:
+  - SimuLand
+query: |
+  AuditLogs
+  | where LoggedByService =~ "Core Directory"
+  | where Category =~ "ApplicationManagement"
+  | where AADOperationType =~ "Assign"
+  | where ActivityDisplayName has_any ("Add delegated permission grant","Add app role assignment to service principal")
+  | mv-expand TargetResources
+  | mv-expand TargetResources.modifiedProperties
+  | extend displayName_ = tostring(TargetResources_modifiedProperties.displayName)
+  | where displayName_ has_any ("AppRole.Value","DelegatedPermissionGrant.Scope")
+  | extend Permission = tostring(parse_json(tostring(TargetResources_modifiedProperties.newValue)))
+  | where Permission contains "RoleManagement.ReadWrite.Directory"
```

Comment:
Same here with usage of "has" versus "contains"

Reply:
Done